optimade/server/entry_collections/entry_collections.py

```diff
@@ -89,10 +91,10 @@ def __init__(
         self.provider_prefix = CONFIG.provider.prefix
         self.provider_fields = [
             field if isinstance(field, str) else field["name"]
-            for field in CONFIG.provider_fields.get(resource_mapper.ENDPOINT, [])
+            for field in CONFIG.provider_fields.get(resource_mapper.ENDPOINT, [])  # type: ignore[call-overload]
```

Comment:
```suggestion
            for field in CONFIG.provider_fields.get(resource_mapper.ENDPOINT, [])
```
I did not need to add #type ignore here.